guides/common/modules/proc_modifying-the-repository-sets.adoc

```diff
@@ -0,0 +1,19 @@
+[id="Modifying_the_Repository_Sets_Status_in_Project{context}"]
```

Comment:
```suggestion
[id="Changing_the_Repository_Sets_Status_in_Project_{context}"]
```

Please also append "-status" to the file name.

I have not found any other instance where we'd use "modify" over "change". Please adjust title, anchor, and file name accordingly.

Reply:
IMHO this has been resolved without actually resolving it. If you intend not to apply my suggestion, at least comment on it.